src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/DefaultTagHelperCompletionService.cs

```diff
@@ -251,5 +254,18 @@ public override IReadOnlyList<CompletionItem> GetCompletionsAt(SourceSpan locati
             snippetText = string.Concat(text, "=\"$1\"");
             return true;
         }
+
+        private Container<string> ResolveAttributeCommitCharacters(IEnumerable<BoundAttributeDescriptor> boundAttributes)
+        {
+            var commitCharacters = new HashSet<string>() { "=" };
+
+            // IsBooleanProperty isn't set
```

Comment:
I'm not entirely sure, but it would explain why we were doing this: https://github.com/dotnet/aspnetcore-tooling/blob/master/src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/DefaultTagHelperCompletionService.cs#L231

Reply:
Hmm, I wonder if it's a deserialization issue where when we deserialize the TagHelpers they don't  get that property set

Reply:
Created a investigation issue for this: https://github.com/dotnet/aspnetcore/issues/21486